modules/channel-telegram/src/backend/client.ts

```diff
@@ -10,24 +10,29 @@ const outgoingTypes = ['text', 'typing', 'image', 'login_prompt', 'carousel']
 
 export const sendEvent = async (bp: typeof sdk, botId: string, ctx: ContextMessageUpdate, args: { type: string }) => {
   // NOTE: getUpdate and setWebhook dot not return the same context mapping
-  const threadId = _.get(ctx, 'chat.id') || _.get(ctx, 'message.chat.id')
-  const target = _.get(ctx, 'from.id') || _.get(ctx, 'message.from.id')
-
-  const payload = _.get(ctx, 'message') || _.get(ctx, 'callback_query')
-  const preview = _.get(ctx, 'message.text') || _.get(ctx, 'callback_query.data')
```

Comment:
Possibly breaking change here but `ctx.callback_query` does not exist in the typings so I'm not sure why it was used here. `ctx.callbackQuery` is how it's written in the typings